internal/broker/response_labels.go

```diff
@@ -35,12 +36,14 @@ func ResponseLabels(op internal.ProvisioningOperation, instance internal.Instanc
 	if enableKubeconfigLabel && !IsOwnClusterPlan(instance.ServicePlanID) {
 		responseLabels[kubeconfigURLKey] = fmt.Sprintf("https://%s/kubeconfig/%s", brokerURL, instance.InstanceID)
 		apiServerUrl, err := kubeconfigBuilder.GetServerURL(instance.RuntimeID)
-		if err != nil {
-			slog.Error(fmt.Sprintf("while getting APIServerURL: %s", err))
-			return responseLabels
+		switch {
+		case err == nil && apiServerUrl != "":
```

Comment:
Should we check `apiServerUrl` or `apiServerUrlKey`?

Reply:
`apiServerUrl`, its value justifies the existence in the map.